{{cookiecutter.project_slug}}/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,98 @@
+name: Deploy my package
+
+on:
+  push:
+    branches:
+      - "master"
+      - "features/**"
+      - "improvements/**"
+      - "bug-fixes/**"
```

Comment:
plz change to "on: pr: " with branches "main" and "master"

Reply:
done